src/main/java/io/tackle/controls/entities/StakeholderGroup.java

```diff
@@ -30,18 +26,15 @@
     @Filterable
     public String description;
 
-    @ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    @ManyToMany(fetch = FetchType.LAZY)
     @JoinTable(
             name = "stakeholdergroup_stakeholders",
             joinColumns = {@JoinColumn(name = "group_id")},
             inverseJoinColumns = {@JoinColumn(name = "stakeholder_id")}
     )
-    @JsonManagedReference
+    @Filterable(filterName = "stakeholders.displayName")
     public List<Stakeholder> stakeholders = new ArrayList<>();
 
-    @Filterable
-    public Integer members;
```

Comment:
Are we planning on just having the ui calculate the number of members for display purposes and apply a filter itself?

Reply:
Yes and no :sweat_smile: 
- Yes: the UI will get the number of members checking the size of the inner `stakeholder` array
- No: the filter won't work on Stakeholders number but on Stakeholder's `displayName` field

Last but not least, the sorting by `members` will be managed by the backend using the query param `sort=stakeholders.size`